From ef3239b7b03a84645d1b64bc031bdc14defd5868 Mon Sep 17 00:00:00 2001
From: J-Pascal Mercier <jp.mercier@gmail.com>
Date: Fri, 27 Jan 2012 18:14:12 -0500
Subject: [PATCH 2/2] BUG FIX - EikonalDataType type array failure

---
 eikonal/data.py |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/eikonal/data.py b/eikonal/data.py
index 9391c65..f71e87f 100644
--- a/eikonal/data.py
+++ b/eikonal/data.py
@@ -41,9 +41,9 @@ class EKTTTable(object):
         try:
             for tname, ttype in tt_dtype:
                 data[tname]
-            self.data = data
+            data = data.astype(self.dtype)
         except ValueError, e:
-            self.data = np.asarray(data, dtype = self.dtype)
+            data = np.asarray(data, dtype = self.dtype)
 
         self.data               = data
         self.data.dtype         = self.dtype
-- 
1.7.8.4

